src/components/queue/QueueListItem/QueueListItem.tsx

```diff
@@ -98,7 +98,7 @@ const QueueListItem: FC<QueueListItemProps> = ({queue, ticket, position}) => {
                                                                onClick={handleMarkReturned}>
                             I&apos;m back
                         </Button>}
-                        {(isTA || isTicketOwner) && !isCompleted &&
+                        {(isTA || isTicketOwner) && !isCompleted && (isTA || isTicketOwner) && 
```

Comment:
is this expected behavior from previous sign me up?

Reply:
could be a good param to have, but a few people asked me for it for student privacy

Reply:
and yes this is how old smu works

Reply:
sg, just wasn't sure